_data/toc.yaml

```diff
@@ -1,3 +1,5 @@
+---
```

Comment:
This is a hack to get Liquid to expand the variable at line 611, and doesn't seem to have any bad side effect.